src/structures/BaseManagerFetchAll.ts

```diff
@@ -0,0 +1,29 @@
+import { FetchOptions, PaginationMeta } from "../common";
+import { BaseManager } from "./BaseManager";
+import { Dict } from "./Dict";
+
+export abstract class BaseManagerFetchAll<T extends unknown[], K, E> extends BaseManager {
+    abstract meta: PaginationMeta;
```

Comment:
Yes, it might just be `public` or `declare`. Still, this would need to be unified with client somehow, as currently I didn't see pages in Client meta.

Reply:
Only the servers endpoint in the client API would need this, the object should be the same.

Reply:
Yes